test/compilable/test19804.d

```diff
@@ -0,0 +1,11 @@
+// https://issues.dlang.org/show_bug.cgi?id=19804
```

Comment:
I think you need `// REQUIRED_ARGS: -g` here.

Reply:
Hmm, we could reproduce by using neted function too, but is it better?

```d
// REQUIRED_ARGS: -g

struct A { float e; }
void foo(A[1]...) { }
```

Reply:
Ok, I see.  disregard my comment.  It's good.